.github/workflows/reusable__format.yaml

```diff
@@ -65,7 +73,10 @@ jobs:
           ref: ${{ inputs.COMMIT_HASH }}
 
       - name: Install Ruby
-        run: sudo apt-get install ruby-full
+        uses: ruby/setup-ruby@v1
```

Comment:
[med​-perf]: I think it is not nescessary to introduce another interpreter when Node is already in place. Why not use https://github.com/marketplace/actions/markdownlint-cli2-action for md formatting?